src/Notifier.js

```diff
@@ -192,12 +216,22 @@ var Notifier = {
         return enabled === 'true';
     },
 
-    setToolbarHidden: function(hidden) {
+     setToolbarHidden: function(hidden, persistent) {
```

Comment:
ok. I suggest you make `true` the default for persistent (and set it to `false` where you call it from `setEnabled`), so that you don't need to change `MatrixToolbar`.